### DISCUSSION TITLE
Add Home Assistant action to "Multi Action" button

### DISCUSSION THREAD

**Discussion by makeitworktech** [cgiesche/streamdeck-homeassistant#284] [Help and Support] [answered]
I would like one button to perform and action on my PC, and after that call a scene in Home Assistant.
I'm trying to find a way to add a Home Assistant action to a multi-action button press. When I drag and drop the Home Assistant Entity action into a multi-action flow, I think it is still expecting a button to be pressed, so the Home Assistant action isn't performed.
I hope you understand the question. See screenshot attached.
![image](https://github.com/user-attachments/assets/999a2692-6bad-4683-b8b6-1919cd2e528d)

**Comment by cgiesche** [comment]
Hi @makeitworktech,

i just added two home-assistant entities to a multi action and it works as expected:
![grafik](https://github.com/user-attachments/assets/8b0433ea-7ea5-4756-b865-b6276f3bd1be)

The plugin does nothing special when it is part of a multiaction. The stream-deck application simply sends a "button pressed" event to all plugins in the multiaction.

**Reply by makeitworktech** [reply, marked as answer]
Ok thanks, I figured it out! I think you need to have something in the appearance field even if you aren't using it as a button. ✅
